crates/rerun_c/src/lib.rs

```diff
@@ -732,15 +732,15 @@ fn rr_log_file_from_path_impl(
     stream: CRecordingStream,
     filepath: CStringView,
     entity_path_prefix: CStringView,
-    timeless: bool,
+    statically: bool,
```

Comment:
I suggest we call this `static_` instead here and everywhere. A suffix `_` would communicate to me that it's only there to prevent a keywaord clash

Reply:
:100: